rhizome/host/lib/spdk_rpc.rb

```diff
@@ -93,6 +93,16 @@ def accel_crypto_key_destroy(name, if_exists = true)
     raise unless if_exists
   end
 
+  def bdev_set_qos_limit(name, rw_ios_per_sec: nil, r_mbytes_per_sec: nil, w_mbytes_per_sec: nil)
+    params = {
+      name: name,
+      rw_ios_per_sec: rw_ios_per_sec || 0,
+      r_mbytes_per_sec: r_mbytes_per_sec || 0,
+      w_mbytes_per_sec: w_mbytes_per_sec || 0
+    }
+    call("bdev_set_qos_limit", params)
+  end
+
```

Comment:
```suggestion
  def bdev_set_qos_limit(name, rw_ios_per_sec: 0, r_mbytes_per_sec: 0, w_mbytes_per_sec: 0)
    params = {
      name:,
      rw_ios_per_sec:,
      r_mbytes_per_sec:,
      w_mbytes_per_sec:
    }
    call("bdev_set_qos_limit", params)
  end

```

Reply:
SPDK expects `0` for "unlimited", but I thought it is easier to understand if we use NULL for unlimited in our database records. So, I use NULL in our logic & just convert it properly when passing it to SPDK. I added a comment.